### DISCUSSION TITLE
Appreciate a backport package for buster

### DISCUSSION THREAD

**Discussion by DIMU4711** [abraunegg/onedrive#1380] [Ideas]
Hello,

I'd like to install the package on debian Buster but there are only files for testing and sid. Would it please be possible to create a backport package for Debian Buster?

Thank you!

DIMU4711

**Comment by abraunegg** [comment]
@DIMU4711 
You will need to take this up with @norbusan for that request.

@norbusan - If you could comment here on the feasibility that would be greatly appreciated.

**Reply by darrepac** [reply]
@norbusan if you can tell us if it will be done anytime soon...if not we will have to use an other path (docker or building)
